autometa/binning/recursive_dbscan.py

```diff
@@ -92,73 +197,72 @@ def recursive_dbscan(table, markers_df, domain, completeness_cutoff,
+            eps *= 10
         if median_completeness == 0:
             clustering_rounds[0] += 1
         if clustering_rounds[0] >= 10:
             break
         if verbose:
             logger.debug(
-                f'EPS: {eps:3.2f} Clusters: {n_clusters:,}'
-                f' Completeness: Median={median_completeness:4.2f} Best={best_median:4.2f}'
+                f"EPS: {eps:3.2f} Clusters: {n_clusters:,}"
+                f" Completeness: Median={median_completeness:4.2f} Best={best_median:4.2f}"
             )
-        eps += step
+        eps += 0.1
     if best_df.empty:
         if verbose:
-            logger.debug('No complete or pure clusters found')
+            logger.debug("No complete or pure clusters found")
         return pd.DataFrame(), table
 
-    completeness_filter = best_df['completeness'] >= completeness_cutoff
-    purity_filter = best_df['purity'] >= purity_cutoff
+    completeness_filter = best_df["completeness"] >= completeness_cutoff
+    purity_filter = best_df["purity"] >= purity_cutoff
     complete_and_pure_df = best_df.loc[completeness_filter & purity_filter]
-    unclustered_df = best_df.loc[~best_df.index.isin(
-        complete_and_pure_df.index)]
+    unclustered_df = best_df.loc[~best_df.index.isin(complete_and_pure_df.index)]
     if verbose:
-        logger.debug(f'Best completeness median: {best_median:4.2f}')
+        logger.debug(f"Best completeness median: {best_median:4.2f}")
     logger.debug(
-        f'clustered: {len(complete_and_pure_df)} unclustered: {len(unclustered_df)}')
+        f"clustered: {len(complete_and_pure_df)} unclustered: {len(unclustered_df)}"
+    )
     return complete_and_pure_df, unclustered_df
 
 
-def run_dbscan(df, eps, dropcols=['cluster', 'purity', 'completeness'],
-               method='dbscan'):
-    """Run clustering on `df` via `method` at provided `eps`.
+def run_hdbscan(df, min_cluster_size, dropcols=["cluster", "purity", "completeness"]):
+    """Run clustering on `df` at provided `eps`.
 
     Notes
     -----
 
-        * documentation for sklearn `DBSCAN <https://scikit-learn.org/stable/modules/generated/sklearn.cluster.DBSCAN.html>`_
+        * reasoning for parameter: `cluster_selection_method <https://hdbscan.readthedocs.io/en/latest/parameter_selection.html#leaf-clustering>`_
+        * reasoning for parameter: `cluster_selection_method <https://hdbscan.readthedocs.io/en/latest/parameter_selection.html#leaf-clustering>`_
         * documentation for `HDBSCAN <https://hdbscan.readthedocs.io/en/latest/index.html>`_
 
     Parameters
     ----------
     df : pd.DataFrame
         Contigs with embedded k-mer frequencies as ['x','y'] columns and optionally 'coverage' column
-    eps : float
+    min_cluster_size : int
         The maximum distance between two samples for one to be considered
         as in the neighborhood of the other. This is not a maximum bound
         on the distances of points within a cluster. This is the most
```

Comment:
Isn't this the minimum number of points that can be in a cluster?

Reply:
Also a few lines below (sorry - this is grey on my view so I can't select), there is mention of DBSCAN, so perhaps we have the wrong description here.

Reply:
Yikes, missed updating this. 🤦‍♂️